experimental/slo/slo_middleware.go

```diff
@@ -0,0 +1,95 @@
+package slo
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/httpclient"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var duration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+	Namespace: "grafana",
```

Comment:
Shouldn't this be:

```suggestion
	Namespace: "plugins",
```

Curiously, we use namespace `plugins` for the instance provider/instance manager:
- https://github.com/grafana/grafana-plugin-sdk-go/blob/36b8fe32821b1740d9fb32a6472287058aee5304/backend/datasource/instance_provider.go#L17-L21
- https://github.com/grafana/grafana-plugin-sdk-go/blob/36b8fe32821b1740d9fb32a6472287058aee5304/backend/instancemgmt/instance_manager.go#L16-L20

but `grafana` for the secure socks proxy :thinking: 

- https://github.com/grafana/grafana-plugin-sdk-go/blob/36b8fe32821b1740d9fb32a6472287058aee5304/backend/proxy/secure_socks_proxy.go#L38-L42